tools/barista/src/utils/fetch-strapi-content.ts

```diff
@@ -87,7 +87,11 @@ async function fetchContent(
 
     const req = request(options, res => {
       res.on('data', data => {
```

Comment:
can this be typed? – everything is any so why toString this is not clear in my opinion. Is it an object or a string or what is the data

Reply:
Yes, I'm sure it can be typed, but with this PR I just want to fix the bug that we currently have when generating the homepage on public build. Please open a follow-up issue for that one. We use `http.request` already since the beginning of this project and it works fine for now.